### PR TITLE
new feature to support C3 nft standard

### PR DESCRIPTION
**Pull request by LevenSton**
new feature to support C3 zombie and DfinityGang nft on plug. plug is the best wallet on IC, hope we can cooperate more in the future.

### CODE REVIEW

File: src/standard_wrappers/nft_standards/ccc.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    return tokensData.map((token) => this.serializeTokenData(token[0], token[1]));
+  }
+
+  async transfer(to: Principal, tokenIndex: number): Promise<void> {
+    const from = await this.agent.getPrincipal();
+    const success:TransferResponse = await this.actor.transferFrom(from, to, BigInt(tokenIndex));
+    if (success.err) {
+      for (let key in success.err)
+        throw new Error(`Error transfering ${key}`);
+    }
```
Comment (tomiir) [inline src/standard_wrappers/nft_standards/ccc.ts:44]: Are you sure this works? Never seen this `:` syntax and just tried on a node console and it throws when trying to access `success` on its own 🤔

File: src/standard_wrappers/nft_standards/ccc.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    });
+  }
+
+  async getUserTokens(principal: Principal): Promise<NFTDetails[]> {
+    const tokensIndexes = await this.actor.getAllNFT(principal);
+    
+    const tokensData = await Promise.all(
+      tokensIndexes.map(async(item) => {
+        const res:GetTokenResponse = await this.actor.getTokenById(item[0])
+        return [res.ok, item[1]]
```
Comment (tomiir) [inline src/standard_wrappers/nft_standards/ccc.ts:31]: same

**Review by tomiir — COMMENTED** [review]
Hey! First of all thanks for the contribution! 🙏 
The code looks very good overall, just have some doubts on some syntax I've never seen!
Else LGTM!

**Comment by LevenSton** [comment]
Hello.  I already change and push the new code.  use suitable vailable and syntax,  our front developer tested, and it can work.  you also can help to confirm if it works:
DfinityGang Canister-id:    2x3lw-tiaaa-aaaah-qcvca-cai
ZombieNFT Canister-id:     iyzne-taaaa-aaaah-qcjga-cai 

Thanks a lot!

**Pull request closed without merging**